design history posts (#364)
DFE-Digital/teacher-cpd-design-history: app/posts/ecf-v2/removing-the-delivery-partner-question.md

---
title: Removing the delivery partner question

> description: "We've tested a few ways to ask school users about the delivery partner they're using for training, when they haven't worked with a delivery partner before. It's not been well understood, and we've chose to remove it."

Reviewer: ```suggestion
description: "We've tested a few ways to ask school users about the delivery partner they're using for training, when they haven't worked with a delivery partner before. It's not been well understood, and we've chosen to remove it."
```

Reviewer: not sure but think it might be past tense

date: 2025-01-22
---

## Why we’re doing this

We created and tested an initial design which asked school users to enter their delivery partner.

We [wanted to ask schools](../asking-for-lead-provider-and-delivery-partner) for this because:

- it helps their lead provider set up training for their ECTs and mentors quickly
- some schools have come to our support desk asking to submit this information, and find it confusing they can't

> - when they've worked with a delivery partner in the most recent academic year, they confirm their delivery partner, and schools can find it confusing they don't when there's been a 2 year gap

Reviewer: ```suggestion
- when they've worked with a delivery partner in the most recent academic year, they confirm their delivery partner, and schools can find it confusing they don't in some scenarios
```

- a school does ultimately decide on their delivery partner, so it better matches reality
- we hoped it would reduce schools having to reach out to delivery partners directly to give information

As we were aware that a lot of schools have the 'same' delivery partner as their appropriate body, we had since designed a journey where we ask users if their appropriate body is also the delivery partner for the ECT they are registering.

We tested this with some users, who all used the 'same' appropriate body and delivery partner, so found the question relatively straightforward.

However, it is also common for appropriate bodies and delivery partners to have differing operating names. In this scenario, we wouldn't be able to ask if the appropriate body is the same organisation as the delivery partner, and we may have to present a different name. This could cause confusion for those users.

> It can also be the cause that sometimes appropriate bodies split into multiple different delivery partners to deliver training too.

Reviewer: ```suggestion
In addition, sometimes appropriate bodies split into multiple different delivery partners to deliver training too.
```
